expression/interpreter/interpreter.go

```diff
@@ -100,6 +101,19 @@ func (ei *ExprInterpreter) InterpretString(strRaw string) ([]byte, error) {
 		return []byte{}, nil
 	}
 
+	// file contents
+	// TODO: make this part of a proper parser
+	if strings.HasPrefix(strRaw, mxscPrefix) {
+		if ei.FileResolver == nil {
+			return []byte{}, errors.New("parser MxscResolver not provided")
+		}
+		fileContents, err := ei.FileResolver.ResolveMxscValue(strRaw[len(filePrefix):])
```

Comment:
The purpose of the FileResolver is just to find files on the disk and load their contents.

The mxsc file parsing logic does not need to be in the FileResolver, the logic for retrieving the file from disk is the same, we just need to apply another function to the loaded bytes, to extract the code.

Simply wrap the json unmarshalling logic in a function ([]byte -> []byte) and call it here.